crates/ruma-state-res/src/event_auth.rs

```diff
@@ -174,6 +184,7 @@ pub fn auth_check<E: Event>(
     }
 
     /*
+    // TODO: Debug
```

Comment:
This is a rather vague todo comment, does it serve a purpose?

Reply:
The code below is disabled because it broke federation, but I don't remember the details

Reply:
What I remember was that there was no clarification if `expected_auth` should equal the incoming events `auth_events` or if the incoming events `auth_events` could be a subset/superset, as in
```rust
if expected_auth != event.auth_events { /* fail event */ }
if !event.auth_events.iter().all(|ev| expected_auth.contains(ev)) { /* fail event */ }
```

Reply:
I still don't like the super-vague comment, can you create an issue instead @timokoesters?

Reply:
I changed the comment, is this better?

Reply:
Yes, thanks!